xml/System.Text.RegularExpressions/GroupCollection.xml

```diff
@@ -1212,9 +1268,9 @@
         <Parameter Name="value" Type="System.Object" Index="0" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="value">The group to locate in the group collection.</param>
+        <summary>Determines the index of a specific group in the group collection.</summary>
+        <returns>The index of the <paramref. name="item" /> if found; otherwise, -1.</returns>
         <remarks>To be added.</remarks>
```

Comment:
```suggestion
This member is an explicit interface member implementation. It can be used only when the <xref:System.Text.RegularExpressions.GroupCollection> instance is cast to an <xref:System.Collections.IList> interface. ```

Reply:
I modified your suggestion a bit to include CDATA portion, otherwise the xref won't work:

```suggestion
        <remarks> 
          <format type="text/markdown"><![CDATA[  

## Remarks

This member is an explicit interface member implementation. It can be used only when the <xref:System.Text.RegularExpressions.GroupCollection> instance is cast to an <xref:System.Collections.IList> interface. 

         ]]></format>
         </remarks>
```